fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rewrite/ScalarOperatorFunctions.java

```diff
@@ -232,6 +234,23 @@ public static long computeYearWeek(long year, long month, long day, int weekBeha
         }
     }
 
+    public static class HashFunctions {
+        private static final long XX_HASH3_64_SEED = 0;
+        public static long hash64(String value) {
+            byte[] data = value.getBytes();
+            LongHashFunction hasher = LongHashFunction.xx3(XX_HASH3_64_SEED);
+            return hasher.hashBytes(data, 0, data.length);
+        }
+    }
+
+    @ConstantFunction.List(list = {
+            @ConstantFunction(name = "xx_hash3_64", argTypes = {VARCHAR}, returnType = BIGINT),
+            @ConstantFunction(name = "xx_hash3_64", argTypes = {CHAR}, returnType = BIGINT)
+    })
+    public static ConstantOperator xxHash64(ConstantOperator input) {
```

Comment:
`xx_hash3_64` is multi-parameters function, e.g.`concat`

Reply:
done